.github/workflows/git-artifacts.yml

```diff
@@ -383,7 +488,7 @@ jobs:
           echo -n "$CODESIGN_PASS" >home/.sig/codesign.pass &&
           git config --global alias.signtool '!sh "/usr/src/build-extra/signtool.sh"'
       - name: Build ${{matrix.arch.bitness}}-bit ${{matrix.artifact.name}}
-        if: env.SKIP != 'true'
+        if: env.SKIP != 'true' && matrix.arch.name != 'arm64'
```

Comment:
At this stage we wouldn't have `sdk32-build-installers`, but we'd need that for ARM64. That problem would be addressed by keeping `arch` as i686 and adding a separate coordinate.

And then we could simply modify _this_ step to pass `--include-arm64-artifacts=$PWD/arm64` only if the `arm64` coordinate is true.

Reply:
Okay, hope to have some time tomorrow towards the end of the day, otherwise this weekend :) to be continued!

Reply:
Sounds good!